### PR TITLE
Feat/run component test

### PR DESCRIPTION
**Pull request by rmoralp**
## Description
Run custom selection of category/component tests with `sui-studio test` command.

Examples: 
- `PATTERN=card/ad/test sui-studio test` runs card/ad component
- `PATTERN={card,shared}/*/test sui-studio test` runs all component tests in card and shared categories
- `PATTERN=card/ad sui-studio test` runs all possible tests with its path contains "card/ad"
- `PATTERN={ad/card/test,user/rating/test}` run tests for those two components

ℹ️ sui-studio already installs `micromatch` because it is a fast-glob sub-dependency, so we could add and use it without increasing install times.

### CODE REVIEW

**Review by nucliweb — APPROVED** [review]

**Review by javiauso — APPROVED** [review]

**Review by jordevo — APPROVED** [review]

**Comment by rmoralp** [comment]
I'll evaluate if we could use glob as a pattern to specify what component tests should run

**Review by midudev — APPROVED** [review]

**Review by andresz1 — APPROVED** [review]

**Comment by andresin87** [comment]
❤️

**Review by andresin87 — APPROVED** [review]

**Review by andresin87 — APPROVED** [review]

**Review by giodelabarrera — APPROVED** [review]
👏🏼

**Review by javiauso — APPROVED** [review]

File: packages/sui-studio/README.md [hunk 1]
```diff
@@ -294,7 +294,41 @@ Add this scripts on your own components project
...
 ```
 
-If you want to execute the tests for some specific categories only use `CATEGORIES` environment variable. It takes a comma separated set of category names (e.g. `CATEGORIES="user,shipping" sui-studio test`)
+```sh
+Usage: sui-studio-test [options]
+
+Options:
+  -H, --headless           Run components tests in CLI, headless mode
+  -W, --watch              Watch mode
+  -T, --timeout <timeout>  Timeout
```
Comment (nucliweb) [inline packages/sui-studio/README.md:303]: In seconds or milliseconds?

**Review by nucliweb — APPROVED** [review]

**Pull request merged**